docs/reference/apis.md

```diff
@@ -89,3 +89,9 @@
 ::: meadowrun.AwsSecret
 
 ::: meadowrun.AzureSecret
+
+## Specifying resource thresholds
+
+::: meadowrun.Threshold
+::: meadowrun.Resources
+::: meadowrun.AllocEC2Instance
```

Comment:
is ManagementConfig part of the public API?

Reply:
yes, but it's not in `meadowrun` package right now, because I figured it's only usable for the configuration of the adjust lambda. I'll add it here as such - unless you have a strong opinion on it should be `meadowrun.ManagementConfig`?

Reply:
On further thought, I'm not making sense, because the same can be said for `Threshold`. So I'll add `ManagementConfig` too.